src/app/view/components/dot-contentlet-editor/dot-edit-contentlet/dot-edit-contentlet.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, Input, EventEmitter, Output, OnInit } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { DotContentletEditorService } from '../services/dot-add-contentlet.service';
+
+@Component({
+    selector: 'dot-edit-contentlet',
+    templateUrl: './dot-edit-contentlet.component.html',
+    styleUrls: ['./dot-edit-contentlet.component.scss']
+})
+export class DotEditContentletComponent implements OnInit {
+    @Input() inode: string;
+    @Output() load: EventEmitter<boolean> = new EventEmitter();
+
+    url: Observable<string>;
+
+    constructor(private dotContentletEditorService: DotContentletEditorService) {}
+
+    ngOnInit() {
+        this.url = this.dotContentletEditorService.edit$;
+    }
+
+    /**
+     * Handle close dialog event
+     *
+     * @memberof DotAddContentletComponent
+     */
+    onClose(): void {
+        this.dotContentletEditorService.clear();
+    }
+
+    /**
+     * Call the keyDown method from the service if exist
+     *
+     * @param {any} $event
+     * @memberof DotAddContentletComponent
+     */
+    onKeyDown($event): void {
+        if (this.dotContentletEditorService.keyDown) {
+            this.dotContentletEditorService.keyDown($event);
+        }
+    }
+
+    /**
+     * Call the load method from the service if exist
+     *
+     * @param {any} $event
+     * @memberof DotAddContentletComponent
+     */
+    onLoad($event): void {
+        if (this.dotContentletEditorService.load) {
+            this.dotContentletEditorService.load($event);
+        }
+    }
```

Comment:
onClose, onKeyDown and onLoad is the same code that the dot-add-content, maybe we can think a way to handle this into the service

Reply:
Ahhh and also, this component will be 100% angular at some point, so they're going to be completely different.

Reply:
Move ```if (this.dotContentletEditorService.load) {``` to the service.